src/Stratis.Bitcoin.Tests/Utilities/AsyncLoopTest.cs

```diff
@@ -28,8 +28,6 @@ public async Task RunOperationCanceledExceptionThrownBeforeCancellationTokenIsCa
 
             await asyncLoop.Run(new CancellationTokenSource(800).Token, TimeSpan.FromMilliseconds(300)).RunningTask;
 
-            this.AssertLog(this.FullNodeLogger, LogLevel.Information, "TestLoop starting.");
```

Comment:
Why delete those?

Reply:
Those are failing on master. 
And it's second time we have tests failing because of string assertions. 
Imho it's a bad idea to verify that a particular log was added

Reply:
I guess it's failing because @MithrilMan moved the `... starting` logs to be logged when the node is actually starting, which is good.
Agree that we shouldn't test logs.

Reply:
Ops my bad I didn't ran tests when I changed the log output, I didn't expected it was tested

Reply:
No problem mate. I also merged it too quickly, my bad.